openmc/tallies.py

```diff
@@ -720,42 +820,45 @@ def get_values(self, scores=[], filters=[], filter_bins=[],
 
         filters : list
-             A list of filter type strings
-             (e.g., ['mesh', 'energy']; default is [])
+            A list of filter type strings
+            (e.g., ['mesh', 'energy']; default is [])
 
         filter_bins : list
-             A list of the filter bins corresponding to the filter_types
-             parameter (e.g., [1, (0., 0.625e-6)]; default is []). Each bin
-             in the list is the integer ID for 'material', 'surface', 'cell',
-             'cellborn', and 'universe' Filters. Each bin is an integer for
-             the cell instance ID for 'distribcell Filters. Each bin is
-             a 2-tuple of floats for 'energy' and 'energyout' filters
-             corresponding to the energy boundaries of the bin of interest.
-             The bin is a (x,y,z) 3-tuple for 'mesh' filters corresponding
-             to the mesh cell of interest. The order of the bins in the list
-             must correspond of the filter_types parameter.
+            A list of the filter bins corresponding to the filter_types
+            parameter (e.g., [1, (0., 0.625e-6)]; default is []). Each bin in
+            the list is the integer ID for 'material', 'surface', 'cell',
+            'cellborn', and 'universe' Filters. Each bin is an integer for the
+            cell instance ID for 'distribcell Filters. Each bin is a 2-tuple of
+            floats for 'energy' and 'energyout' filters corresponding to the
+            energy boundaries of the bin of interest.  The bin is a (x,y,z)
+            3-tuple for 'mesh' filters corresponding to the mesh cell of
+            interest. The order of the bins in the list must correspond of the
+            filter_types parameter.
 
         nuclides : list
-             A list of nuclide name strings
-             (e.g., ['U-235', 'U-238']; default is [])
+            A list of nuclide name strings
+            (e.g., ['U-235', 'U-238']; default is [])
 
         value : str
-             A string for the type of value to return  - 'mean' (default),
-             'std_dev', 'rel_err', 'sum', or 'sum_sq' are accepted
+            A string for the type of value to return  - 'mean' (default),
+            'std_dev', 'rel_err', 'sum', or 'sum_sq' are accepted
 
         Returns
         -------
-             A scalar or NumPy array of the Tally data indexed in the order
-             each filter, nuclide and score is listed in the parameters.
+        float or ndarray
+            A scalar or NumPy array of the Tally data indexed in the order
+            each filter, nuclide and score is listed in the parameters.
 
         Raises
         ------
-             ValueError : An error when this routine is called before the Tally
-             is populated with data by the StatePoint.read_results() routine.
+        ValueError
+            When this routine is called before the Tally is populated with data
+            by the StatePoint.read_results() routine.
```

Comment:
You might also note that a ``ValueError`` will be thrown if the input parameters do not correspond to the Tally's attributes (*e.g.*, if the score(s) do not match those in the ``Tally``)